terraform/environments/ci/files/decrypt.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+unset IFS
+out_path=${1:-/out/secrets.env}
+rm "$out_path" 2> /dev/null
+for line in $(env | egrep '^KMS_'); do
+	kms_key="${line%%=*}"
+	key=${kms_key/KMS_/}
+	encrypted_value=${line#*=}
```

Comment:
It is bash variable substitution %%=* says "strip the equals sign and anything after it" and #*= says "remove the equals sign and anything before it". When using bash there is little need for things like sed/awk. 

I am only injecting the bash binary into the container so KISS.